docs/07. media library/03. integrating in your module.md

```diff
@@ -51,6 +51,15 @@ Symfony FormType class:
 use Backend\Modules\MediaLibrary\Domain\MediaGroup\MediaGroupType;
 ```
 
+Or if you only want a single media item to be connected you can use the SingleMediaGroupType
+```php
+use Backend\Modules\MediaLibrary\Domain\MediaGroup\SingleMediaGroupType;
+```
+
+This will set the options minimum_items and maximum_items to 1 and add the constraint to enforce it.
+These options are also available on the MediaGroupType if you would want a different range.
+To help the end user there is also some client side validation that will indicate when more items need to be connected or too many items are connected.
+
```

Comment:
The description looks fine but the example below doesn't match the form type you just described. I would put this text under the example and add an example of the SingleMediaGroupType.

Reply:
fixed in a02f2e551